csrc/ops/indexing.cpp

```diff
@@ -316,39 +316,22 @@ TensorView* takeAlongAxis(TensorView* inp, TensorView* index, int64_t dim) {
   return out_tensor->as<TensorView>();
 }
 
-TensorView* preprocessGroupedMatmulInputSf(
-    TensorView* input,
-    TensorView* input_offsets,
-    TensorView* output_offsets,
+std::vector<IterDomain*> layoutAllocationDomain(
```

Comment:
these are just mechanical changes, since in a later PR `layoutAllocationDomain` is used separately for replay.